.drone.yml

```diff
@@ -8,6 +9,29 @@ platform:
   version: 1809
 
 steps:
+- name: docker-build
```

Comment:
Was this always a missing step in the `.drone.yml` or is there a specific reason why we need this now?Was this always a missing step in the `.drone.yml` or is there a specific reason why we need this now?

Reply:
This was originally missing so docker builds were not exercised on PR. That means that those weren't being validated for any breaking changes on PR.